From 5ee886fd3cb8530e33b4d06b1061c15c5d1af3b1 Mon Sep 17 00:00:00 2001
From: Jan Kiszka <jan.kiszka@siemens.com>
Date: Sun, 6 Sep 2020 17:30:55 +0200
Subject: [PATCH 09/15] mm: vmalloc: Export __get_vm_area_caller

Needed by Jailhouse so far, as replacement of __get_vm_area. Should be
solved there eventually (via removal of JAILHOUSE_BORROW_ROOT_PT), then
this can be dropped again.

Upstream-Status: Pending

Signed-off-by: Jan Kiszka <jan.kiszka@siemens.com>
(cherry picked from commit b5ca803f2e5dcb1905a95915d6ffa34bf53a2db7)
Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
---
 mm/vmalloc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mm/vmalloc.c b/mm/vmalloc.c
index dc4913366d2f85..5121265e412a9a 100644
--- a/mm/vmalloc.c
+++ b/mm/vmalloc.c
@@ -2533,6 +2533,7 @@ struct vm_struct *__get_vm_area_caller(unsigned long size, unsigned long flags,
 	return __get_vm_area_node(size, 1, PAGE_SHIFT, flags, start, end,
 				  NUMA_NO_NODE, GFP_KERNEL, caller);
 }
+EXPORT_SYMBOL_GPL(__get_vm_area_caller);
 
 /**
  * get_vm_area - reserve a contiguous kernel virtual area
-- 
2.40.1

